src/background/services/balances/BalanceAggregatorService.ts

```diff
@@ -89,10 +89,18 @@ export class BalanceAggregatorService implements OnLock, OnUnlock {
       .map(({ value }) => value);
 
     const networksWithChanges = updatedNetworks
-      .filter(
-        ({ chainId, networkBalances }) =>
-          !isEqual(networkBalances, this.balances[chainId])
-      )
+      .filter(({ chainId, networkBalances }) => {
```

Comment:
This fixes a bug I've noticed in the P-Chain balances fetching.

Basically, when user has multiple accounts derived, we would always detect changes here because we only poll for the active account and we'd have balances for multiple addresses cached.